### PR TITLE
Fix searchbar

### PR DESCRIPTION
**Pull request by Karim94**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/437/mobile-search-fix-spacing-and-nav-bar
https://app.clubhouse.io/peerio/story/443/search-clear-icon
#### Testing instructions  

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: app/components/helpers/icons.js [hunk 1]
```diff
@@ -153,16 +151,16 @@ const icons = {
...
 
-    iconAddChat(onPress) {
+    iconImage(source, onPress) {
         const width = vars.iconSize;
         const height = width;
         const padding = vars.iconPadding;
         return (
             <TouchableOpacity
-                style={{ padding }}
+                style={{ padding, opacity: 0.54 }}
```
Comment (seavan) [inline app/components/helpers/icons.js]: very specific, should go to vars.

File: app/components/layout/tabs.js [hunk 2]
```diff
@@ -54,6 +55,7 @@ export default class Tabs extends SafeComponent {
     }
 
     renderThrow() {
+        if (uiState.keyboardHeight) return null;
```
Comment (seavan) [inline app/components/layout/tabs.js]: this doesn't break anything anywhere else?

Comment (Karim94) [reply app/components/layout/tabs.js]: Currently, tabs are only being used in layout-main and in file-view. We do not have any text inputs in any of those two views.

Comment (seavan) [reply app/components/layout/tabs.js]: layout-main contains all the views and routes in the logged-in state, though )

**Comment by seavan** [comment]
@ebilg ready for testing

**Pull request merged**